.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
> We really can't run the foxbox daemon as root... Instead of using avahi_client_set_host_name() did you try to add a cname record like in http://www.avahi.org/wiki/Examples/PythonPublishAlias ? This works for me without being root.

Nope haven't tried it, seems it's too naive and doesn't support collision resolution as we wanted.

> You don't even need to use dbus hopefully, just getting a group and adding the record should work.

This python script uses DBus/Avahi Daemon for managing entry groups, and if I'm not mistaken, not using DBus just means not using Avahi Daemon at all (and implement everything on our own instead) if we don't want troubles as it will basically mean having two "daemons" who is in charge of managing mDNS host.

So let me dig a bit, to understand if we can get rid of "root" requirements with option 3 or option 2 from message above is the only way.

Ah and just curious why we want to avoid using DBus by all means - is it heavy\buggy or ..?